include/systems/system.h

```diff
@@ -113,6 +113,22 @@ class System : public ReferenceCountedObject<System>,
           const std::string & name,
           const unsigned int number);
 
+  /**
+   * Special functions.
+   * - The copy constructor and assignment operator were previously
+   *   declared as private libmesh_not_implemented() functions, this
+   *   is the C++11 way of achieving the same effect.
+   * - The System holds references to Mesh and EquationSystems
+   *   objects, therefore it can't be default move-assigned.
+   * - This class _can_ be default move constructed.
+   * - The destructor throws an error if libMesh::closed()
```

Comment:
This should say something like "dies with an error in dbg/devel modes"?  We can't throw from a destructor and we don't bother to double-check in opt mode.

Reply:
Good point, I will change those docs.

Reply:
Fixed in fd6ac4977